nerfstudio/process_data/colmap_utils.py

```diff
@@ -449,9 +450,17 @@ def colmap_to_json(
         frames.append(frame)
 
     if set(cam_id_to_camera.keys()) != {1}:
-        raise RuntimeError("Only single camera shared for all images is supported.")
-    out = parse_colmap_camera_params(cam_id_to_camera[1])
-    out["frames"] = frames
+        cameras = {k: parse_colmap_camera_params(v) for k, v in cam_id_to_camera.items()}
+        model = list(set(x["camera_model"] for x in cameras.values()))
```

Comment:
What if we have multiple camera models? Shall we also raise an exception here?

Maybe we should assert this assumption since colmap only supports one camera model at a time.